eos/utils/kmatrix_TEST.cc

```diff
@@ -0,0 +1,260 @@
+/*
+ * Copyright (c) 2021 Méril Reboud
+ *
+ * This file is part of the EOS project. EOS is free software;
+ * you can redistribute it and/or modify it under the terms of the GNU General
+ * Public License version 2, as published by the Free Software Foundation.
+ *
+ * EOS is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+ * Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+
+#include <test/test.hh>
+#include <eos/utils/parameters.hh>
+#include <eos/utils/kmatrix-impl.hh>
+#include <eos/utils/complex.hh>
+#include <iostream>
+#include <memory>
+
+#include <cmath>
+
+
+using namespace test;
+using namespace eos;
+
+
+///////////////////////
+/// Simplest K matrix, 1 channel, 1 resonance
+///////////////////////
+struct PPchan11 : public KMatrix<1,1>::Channel
+{
+    PPchan11(std::string name, double m1, double m2, unsigned N_orbital, const Parameters & p) : Channel(name, m1, m2, N_orbital, {{
```

Comment:
Please treat the base class constructor as an initialization, and put it onto an (indented) line of its own.